search-api/src/search_api/resources/v1/businesses/search.py

```diff
@@ -72,14 +73,17 @@ def facets():  # pylint: disable=too-many-branches, too-many-locals
                     identifier = param
                 elif param := _parse_url_param(SolrField.BN.value, item):
                     bn = param  # pylint: disable=invalid-name
+                elif param := _parse_url_param(SolrField.GOOD_STANDING.value, item):
+                    good_standing = param
         if not value:
             return jsonify({'message': "Expected url param 'query' to have 'value:<string>'."}), HTTPStatus.BAD_REQUEST
         # clean query values
         query = {
             'value': Solr.prep_query_str(value),
             SolrField.NAME_SINGLE.value: Solr.prep_query_str(name),
             SolrField.IDENTIFIER_Q.value: Solr.prep_query_str(identifier),
-            SolrField.BN_Q.value: Solr.prep_query_str(bn)
+            SolrField.BN_Q.value: Solr.prep_query_str(bn),
+            SolrField.GOOD_STANDING.value: Solr.prep_query_str(good_standing)
```

Comment:
Since this is a boolean field we would want to add this as a category. I think its better to remove this for now since there isn't any business value to searching by goodStanding